services/test-validators.js

```diff
@@ -93,7 +93,9 @@ const isPercentage = Joi.alternatives().try(
   isDecimalPercentage
 )
 
-const isFileSize = withRegex(/^[0-9]*[.]?[0-9]+\s(B|kB|MB|GB|TB|PB|EB|ZB|YB)$/)
+const isFileSize = withRegex(
+  /^[0-9]*[.]?[0-9]+\s(B|kB|KB|MB|GB|TB|PB|EB|ZB|YB)$/
+)
```

Comment:
had to account for some pedantry here since the random target I grabbed had a file size in the kilobytes, but Spiget deals with base 2 as opposed to base 10, and in base 2 the leading `K` for kilobytes is capitalized